javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,38 @@
+function translateOneByOne() {
+
+
+
+    moveElement(document.querySelector("li:nth-child(1)"), { x: 20, y: 300 }).then(() => {
+        console.log("Red target has been moved");
+
+
+        moveElement(document.querySelector("li:nth-child(2)"), { x: 400, y: 300 }).then(() => {
+            console.log("Blue target has been moved");
+
+            moveElement(document.querySelector("li:nth-child(3)"), { x: 400, y: 20 }).then(() => {
+                console.log("Green target has been moved");
+
+            }).catch((err) => console.log('Red target went wrong'));
+
+        }).catch((err) => console.log('Blue target went wrong'));
+
+    }).catch((err) => console.log('Green target went wrong'));
```

Comment:
Looks good, but consider simplifying this very long statement. Use variables, async/await and try/catch

Reply:
I've tried to do so, commited changes.